apps/admin/backend/schema.sql

```diff
@@ -95,6 +95,7 @@ create table write_ins (
   official_candidate_id text,
   write_in_candidate_id varchar(36),
   is_invalid boolean not null default false,
+  is_manually_created boolean not null default false,
```

Comment:
Also considering naming this `is_undetected`, to indicate we didn't detect the write-in, but manually created it. Does one name seem more in line with our naming conventions?

Reply:
No strong preference myself.

Reply:
Changing to `is_undetected` as that seems more idiomatic here